package/sdbus-cpp/sdbus-cpp.mk

```diff
@@ -0,0 +1,26 @@
+################################################################################
+#
+# SDBUS_CPP
+#
+################################################################################
+
+SDBUS_CPP_VERSION = 0.8.2
+SDBUS_CPP_SITE = $(call github,Kistler-Group,sdbus-cpp,v$(SDBUS_CPP_VERSION))
+SDBUS_CPP_LICENSE = LGPL-2.1
+SDBUS_CPP_LICENSE_FILES = COPYING
+SDBUS_CPP_DEPENDENCIES = host-pkgconf systemd
+SDBUS_CPP_CONF_OPTS += \
+	-DBUILD_CODE_GEN=OFF \
+	-DBUILD_DOC=OFF \
+	-DBUILD_TESTS=OFF \
+	-DBUILD_LIBSYSTEMD=OFF
+SDBUS_CPP_INSTALL_STAGING = YES
+	
+HOST_SDBUS_CPP_DEPENDENCIES = host-pkgconf
+HOST_SDBUS_CPP_CONF_OPTS += \
+	-DBUILD_CODE_GEN=ON \
+	-DBUILD_DOC=OFF \
+	-DBUILD_TESTS=OFF \
+	-DBUILD_LIBSYSTEMD=ON
+
+$(eval $(cmake-package))
```

Comment:
$(eval $(host-cmake-package))

You need that if you are building the host tool

Reply:
Thanks!